packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -46,16 +47,25 @@ function getImgNodeKey(elementId) {
   return `${IMG_NODE}_${elementId}`;
 }
 
-export function getMediaBaseColor(resource, onBaseColor) {
-  const { type, src, poster } = resource;
-  setOrCreateImage({
-    src: type === 'video' ? poster : src,
-    width: 10,
-    height: 'auto',
-  }).then(
-    (color) => onBaseColor(color),
-    () => onBaseColor([255, 255, 255]) // Fallback color is white.
-  );
+export async function getMediaBaseColor(src) {
+  if (!src) {
+    return Promise.reject(new Error('No source to image'));
+  }
+  let color;
+  try {
+    color = await setOrCreateImage({
+      src,
+      width: 10,
+      height: 'auto',
+    });
+  } catch (error) {
+    // Known error of color thief with white only images.
```

Comment:
Do I understand this comment correctly in that white-only images result in TypeError?

Reply:
It does, other kinds of errors are throw still.